src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--1">
```

Comment:
<img width="927" alt="image" src="https://user-images.githubusercontent.com/99403198/234550665-f8716502-2cf6-4381-b642-389072585998.png">
```suggestion
    </div>
    
    <div class="stars stars--1">
```